CONTRIBUTING.md

```diff
@@ -39,6 +39,52 @@ When you send a pull request, make sure that the `index.json` file is up-to-date
 with the changes in the `payload-examples/api.github.com/` folder, otherwise the
 tests will fail.
 
+## Creating a new schema
+
+The webhook schemas are [JSON schemas](https://json-schema.org/) that are manually created using example payloads for webhook events.
+
+- Create a new folder for the event, if it doesn't already exist
+- Create a new JSON file named `<action>.schema.json` if the event has an `action` property, otherwise name it `event.schema.json`
+- The `$id` property's value should be in the format `<eventName>$action`, where `<eventName>` is the name of the event, and `action` is the value of the `action` property of the webhook payload or simply `event` if there is no `action` property for the event
```

Comment:
For consistency I think we should use `<action>` here:

```suggestion
- The `$id` property's value should be in the format `<eventName>$<action>`, where `<eventName>` is the name of the event, and `<action>` is the value of the `action` property of the webhook payload or simply `event` if there is no `action` property for the event
```

nit: though I'd probably suggest rephrasing this to be something like:

> The `$id` property should start with the event name, followed by the `action` name separated with `$` if the event has an `action` property, e.g. `pull_request$created`

Reply:
I went with your rephrased version